dev/components/recordPopup/component.js

```diff
@@ -59,14 +54,17 @@
     };
 
     vm.sendRecordClick = function (name) {
-      name = name || 'no name';
-      uploadService.sendFile(
-        '/api/video',
-        vm.recordedData.video,
-        name + '.wmv'
-      ).then(function (res) {
+      save(name).then(function (res) {
         $location.path('template/?video=' + res.data.id);
       });
     };
+
+    function save(name) {
+      name = name || 'no name';
+      return uploadService.sendFile(
+        '/api/video',
+        vm.recordedData.video,
+        name + '.wmv');
```

Comment:
This extension is wrong. Please verify the information https://github.com/collab-project/videojs-record